### PR TITLE
Grok processor report match errors by default

### PR DESCRIPTION
**Pull request by yyuuttaaoo**

### CODE REVIEW

File: pluginmanager/global_config.go [hunk 1]
```diff
@@ -76,12 +75,8 @@ func newGlobalConfig() (cfg GlobalConfig) {
 		FlushIntervalMs:          3000,
 		DefaultLogQueueSize:      1000,
 		DefaultLogGroupQueueSize: 4,
+		LogtailSysConfDir:        ".",
 		DelayStopSec:             300,
 	}
-	if runtime.GOOS == "windows" {
```
Comment (messixukejia) [inline pluginmanager/global_config.go:81]: windows、liux都测试影响了吗

File: pluginmanager/plugin_manager.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ var DisabledLogtailConfigLock sync.Mutex
 var DisabledLogtailConfig = make(map[string]*LogstoreConfig)
 
 // StatisticsConfigJson, AlarmConfigJson
-var BaseVersion = "0.1.0"
+var BaseVersion = "1.3.1"
```
Comment (messixukejia) [inline pluginmanager/plugin_manager.go:44]: 这个要不要放到一个单独文件里？

**Review by messixukejia — CHANGES_REQUESTED** [review]

File: pluginmanager/global_config.go (hunk 1 above)
Comment (yyuuttaaoo) [reply pluginmanager/global_config.go:81]: linux无问题

File: plugins/processor/grok/processor_grok.go [hunk 3]
```diff
@@ -346,8 +346,8 @@ func init() {
 			IgnoreParseFailure:  true,
 			KeepSource:          true,
 			NoKeyError:          false,
-			NoMatchError:        false,
-			TimeoutError:        false,
+			NoMatchError:        true,
```
[the remark this answers is not in the document]
Comment (yyuuttaaoo) [reply plugins/processor/grok/processor_grok.go:349]: ok

File: pluginmanager/plugin_manager.go (hunk 2 above)
Comment (yyuuttaaoo) [reply pluginmanager/plugin_manager.go:44]: Refined as described in this article: https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications

**Review by messixukejia — APPROVED** [review]

**Pull request merged**